vendor/foxy/CMakeLists.txt

```diff
@@ -167,7 +179,8 @@ option(BUILD_TESTING "Build Foxy Testsuite" OFF)
 if (BUILD_TESTING)
     include(CTest)
 
-    find_package(Catch2 CONFIG REQUIRED)
```

Comment:
Given I'm swapping out the certify library, I figured it would be good to have foxy's tests enabled. This necessitates grabbing catch2 with FetchContent.